NBitcoin/RPC/RPCClient.cs

```diff
@@ -1065,6 +1065,26 @@ public int GetBlockCount()
 			return (int)(await SendCommandAsync(RPCOperations.getblockcount).ConfigureAwait(false)).Result;
 		}
 
+		public MemPoolInfo GetMemPool()
+		{
+			return this.GetMemPoolAsync().GetAwaiter().GetResult();
+		}
+
+		public async Task<MemPoolInfo> GetMemPoolAsync()
+		{
+			var response = await SendCommandAsync(RPCOperations.getmempoolinfo);
+
+			return new MemPoolInfo()
+			{
+				Size = Int32.Parse((string)response.Result["size"]),

```

Comment:
Can you pass `CultureInfo.InvariantCulture` as second parameter of those `Parse` methods? Else it will explode for some users.

Reply:
Of course! There are 14 other `Parse` (mostly `uint`) without the CultureInfo provided in `RPCClient.cs`, should I add them there as well or just focus on what was added?